app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    if command[0] == "mv" and len(command) == 3:
+        try:
+            mv, file1_name, destination = command
+            if destination.endswith("/"):
+                raise ValueError("Enter correct path and file name!")
```

Comment:
The task description specifies that if the destination path ends with a slash, it should be considered as a directory. Therefore, raising a ValueError in this case is incorrect. You should modify this logic to handle such paths appropriately .